contribute/index.html

```diff
@@ -12,10 +12,10 @@
             <strong>Hoodie wouldn’t be where it is today without the people</strong> who supported it by dedicating their time, skills, knowledge, by giving talks about it and supporting the project financially.
         </p>
         <p>
-            Here at Hoodie’s, there’s always a lot of work to do for everyone, for example for people who do programming, design, writing, organising and many more. <strong>We &#10084; the people who are involved in this project</strong>, and we’d love to have you on board.
+            Here at Hoodie, there’s always a lot of work to do for everyone, for example for people who do programming, design, writing, organising and many more. <strong>We &#10084; the people who are involved in this project</strong>, and we’d love to have you on board.
         </p>
         <p>
-            If you want to take a first look at <strong>what we've been working recently</strong>, a good start may be to look at our <a href="http://gr2m.github.io/milestones/" target="_blank">Milestones</a>, the <a href="http://espy.github.io/ubersicht/#hoodiehq" target="_blank">Ubersicht</a> of our activity on Hoodie's <a href="http://github.com/hoodiehq" target="_blank">GitHub</a> activity, — and of course our <a href="/blog">Blog</a> and our <a href="http://twitter.com/hoodiehq" target="_blank">Twitter</a> always offer the latest information as well.
+            If you want to take a first look at <strong>what we've been working recently</strong>, a good start may be to look at our <a href="http://gr2m.github.io/milestones/" target="_blank">Milestones</a>, the <a href="http://espy.github.io/ubersicht/#hoodiehq" target="_blank">Ubersicht</a> of our activity on Hoodie's <a href="http://github.com/hoodiehq" target="_blank">GitHub</a> activity, — and of course our <a href="/blog">blog</a> and our <a href="http://twitter.com/hoodiehq" target="_blank">Twitter</a> always offer the latest information as well.
```

Comment:
Okay, I think I figured out how to make the correction. The sentence should now include the missing word "on". Thank you, Charlotteis!